event_routing_backends/processors/xapi/transformer.py

```diff
@@ -100,6 +102,14 @@ def get_actor(self):
             )
         return agent
 
+    @openedx_filter(filter_type="event_routing_backends.processors.xapi.transformer.xapi_transformer.get_verb")
+    def get_verb(self):
+        """
+        This intercepts the super verb value or the attribute class `_verb` in order to allow the openedx
+        filters implementation.
+        """
+        return super().get_verb() if hasattr(super(), "get_verb") else self._verb  # pylint: disable=no-member
```

Comment:
yes, I have identified two cases

1. there are transformers that just inherits from one class `XApiTransformer` like [CompletionCreatedTransformer](https://github.com/openedx/event-routing-backends/blob/master/event_routing_backends/processors/xapi/event_transformers/completion_events.py#L12C7-L12C35)
2. there are transformers that inherits from multiple classes `XApiTransformer, XApiVerbTransformerMixin` like [ProblemEventsTransformer](https://github.com/openedx/event-routing-backends/blob/master/event_routing_backends/processors/xapi/event_transformers/problem_interaction_events.py#L128C7-L128C30) indeed this inherits from `BaseProblemsTransformer` but that class inherits from `XApiTransformer` and `XApiVerbTransformerMixin`

For the second case there is no problem  since the class `XApiVerbTransformerMixin` has that [method](https://github.com/openedx/event-routing-backends/blob/84ae01a197ced05bed84a8e28f0fd6989ee46944/event_routing_backends/processors/xapi/transformer.py#L180) however in the first case the  `XApiTransformer` doesn't have a parent class and the sub-classes doesn't implement another parent class with that method therefore the `super().get_verb()` won't exist